.github/workflows/tox-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Tox Tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.6', '3.7', '3.8', '3.9']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox~=3.28 'tox-gh-actions<3'
```

Comment:
Why do we need to pin these?

Reply:
Tox v4 is a complete rewrite of the package and requires Python 3.7. 

Also I didn't want to have to check all these https://tox.wiki/en/latest/upgrading.html

The `tox-gh-actions`  restriction, is just for tox 3 compat

Reply:
Fair enough :+1: